CollAction/Controllers/ManageController.cs

```diff
@@ -61,16 +62,29 @@ public async Task<IActionResult> Index()
                 Username = user.UserName,
                 Email = user.Email,
                 StatusMessage = StatusMessage,
-                NewsletterSubscription = await _newsletterSubscriptionService.IsSubscribedAsync(user.Email),
-                ProjectsParticipated = (await _context.ProjectParticipants
-                                                      .Where(part => part.UserId == user.Id)
-                                                      .Include(part => part.Project)
-                                                      .ToListAsync()).Select(part => part.Project),
-                ProjectsCreated = await _context.Projects.Where(proj => proj.OwnerId == user.Id && proj.Status != ProjectStatus.Deleted).ToListAsync()
+                NewsletterSubscription = await _newsletterSubscriptionService.IsSubscribedAsync(user.Email)
             };
             return View(model);
         }
 
+        [HttpGet]
+        public async Task<JsonResult> MyProjects()
```

Comment:
For methods requiring a user, perhaps it'd be a good idea to add the [Authorize] and [ValidateAntiForgeryToken] tags.

Reply:
Fixed